src/test/scala/com/microsoft/hyperspace/SparkInvolvedSuite.scala

```diff
@@ -31,6 +33,7 @@ trait SparkInvolvedSuite extends BeforeAndAfterAll with BeforeAndAfter {
   protected lazy val spark: SparkSession = SparkSession
     .builder()
     .master(s"local[$numParallelism]")
+    .config(HYPERSPACE_EVENT_LOGGER_CLASS_KEY, "com.microsoft.hyperspace.MockEventLogger")
```

Comment:
for reviewers: setting this config at the beginning of tests because it's a lazy val in a object EventLogger class, and hence it is initialized only once at the beginning of the test suite.

I tried setting this within a test but since the lazy val got initialized before this config could take effect, it did not work.